.prow/config.yaml

```diff
@@ -209,97 +209,6 @@ presubmits:
     branches:
     - ^v0\.(3|4)-branch$
 
-  - name: test-end-to-end-batch
-    decorate: true
-    always_run: true
-    spec:
-      volumes:
-      - name: service-account
-        secret:
-          secretName: feast-service-account
-      containers:
-      - image: maven:3.6-jdk-11
-        command: ["infra/scripts/test-end-to-end-batch.sh"]
-        resources:
-          requests:
-            cpu: "6"
-            memory: "6144Mi"
-        volumeMounts:
-        - name: service-account
-          mountPath: "/etc/service-account"
-    skip_branches:
-    - ^v0\.(3|4)-branch$
-
-  - name: test-end-to-end-batch-fs-update
-    decorate: true
-    always_run: false
-    spec:
-      volumes:
-        - name: service-account
-          secret:
-            secretName: feast-service-account
-      containers:
-        - image: maven:3.6-jdk-11
-          command: ["infra/scripts/test-end-to-end-batch.sh", "-m", "fs_update"]
-          resources:
-            requests:
-              cpu: "6"
-              memory: "6144Mi"
-          volumeMounts:
-            - name: service-account
-              mountPath: "/etc/service-account"
-    skip_branches:
-      - ^v0\.(3|4)-branch$
-
-  - name: test-end-to-end-batch-java-8
```

Comment:
I would prefer this to be kept. There might still be users for Feast 0.3/0.4.

Reply:
Reverted.